cloud/scope/cluster_test.go

```diff
@@ -713,3 +713,204 @@ func TestDeleteSubnet(t *testing.T) {
 		})
 	})
 }
+
+func TestCreateLoadBalancer(t *testing.T) {
+	var (
+		mockvpc  *mock.MockVpc
+		mockCtrl *gomock.Controller
+	)
+
+	setup := func(t *testing.T) {
+		t.Helper()
+		mockCtrl = gomock.NewController(t)
+		mockvpc = mock.NewMockVpc(mockCtrl)
+	}
+	teardown := func() {
+		mockCtrl.Finish()
+	}
+
+	vpcCluster := infrav1beta1.IBMVPCCluster{
+		Spec: infrav1beta1.IBMVPCClusterSpec{
+			ControlPlaneLoadBalancer: &infrav1beta1.VPCLoadBalancerSpec{
+				Name: "foo-load-balancer",
+			},
+		},
+		Status: infrav1beta1.IBMVPCClusterStatus{
+			Subnet: infrav1beta1.Subnet{
+				ID: core.StringPtr("foo-subnet-id"),
+			},
+		}}
+	expectedOutput := &vpcv1.LoadBalancer{
```

Comment:
This is used only in one test case, hence I don't see point in declaring this globally!

Reply:
Fixed.